config/config_params.go

```diff
@@ -140,8 +140,8 @@ type Config struct {
 	PrometheusMetricsEnabled bool `config:"bool;false"`
 	PrometheusMetricsPort    int  `config:"int(0,65535);9091"`
 
-	FailsafeInboundHostPorts  []uint16 `config:"port-list;22;die-on-fail"`
-	FailsafeOutboundHostPorts []uint16 `config:"port-list;2379,2380,4001,7001;die-on-fail"`
+	FailsafeInboundHostPorts  []ProtoPort `config:"port-list;tcp:22,udp:68;die-on-fail"`
+	FailsafeOutboundHostPorts []ProtoPort `config:"port-list;tcp:2379,tcp:2380,tcp:4001,tcp:7001,udp:53,udp:67;die-on-fail"`
```

Comment:
Sorry for making you sad but we discussed and you said you can live-with so I plan to leave as-is.